lib/kafka_ex/server_0_p_10_and_later.ex

```diff
@@ -40,6 +39,7 @@ defmodule KafkaEx.Server0P10AndLater do
 
   defdelegate kafka_server_consumer_group_metadata(state), to: Server0P8P2
   defdelegate kafka_server_update_consumer_metadata(state), to: Server0P8P2
+  defdelegate update_consumer_metadata(state, num, error_code), to: Server0P8P2
```

Comment:
This doesn't seem to be necessary for the snappyer change.  Please make it a separate pull request.